src/Twig/Extension/ViewExtension.php

```diff
@@ -33,6 +33,29 @@ class ViewExtension extends AbstractExtension
     public function getFunctions(): array
     {
         return [
+            new TwigFunction(
+                'cell',
+                function ($context, string $name, array $data = [], array $options = []) {
+                    $view = $context['_view'];
+                    $cell = $view->cell($name, $data, $options);
+
+                    $original = [];
+                    foreach ($view->getDefaultConfig() as $config => $default) {
```

Comment:
Why is using `getDefaultConfig()` necessary instead of just using `getConfig()`?

Reply:
I used it to get the list of TwigView-specific keys, otherwise I have to hard-code it deep in this extension code.

Reply:
`View` itself doesn't have any config, so any config you get is TwigView specific.

Reply:
Ah, I see. You can get the entire config back. I missed how to do that.